### PR TITLE
kernel-arch-complete.sh: Add email reports for tree cip-gitlab

### PR DESCRIPTION
**Pull request by patersonc**
Only report results to CIP directly as this tree is mainly used for
release candidate testing.

Signed-off-by: Chris Paterson <chris.paterson2@renesas.com>

### CODE REVIEW

File: scripts/kernel-arch-complete.sh [hunk 1]
```diff
@@ -103,6 +103,20 @@ fi
...
         curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'",  "report_type": "test", "plan": "v4l2-compliance-uvc", "send_to": ["cip-dev+int+1680+2922216858170285886@lists.cip-project.org", "kernel-build-reports@lists.linaro.org", "kernelci-results@groups.io"], "format": ["txt"], "delay": 3600}' ${API}/send
+    elif [ "$TREE_NAME" == "cip-gitlab" ]; then
+        echo "Sending results for CIP (GitLab) tree"
+        curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'", "build_report": 1, "format": ["txt"], "send_to": ["cip-dev+int+1680+2922216858170285886@lists.cip-project.org"], "delay": 60}' ${API}/send
+        curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'",  "report_type": "test", "plan": "baseline", "send_to": ["cip-dev+int+1680+2922216858170285886@lists.cip-project.org"], "format": ["txt"], "delay": 1800}' ${API}/send
```
Comment (gctucker) [inline scripts/kernel-arch-complete.sh:109]: How about `baseline-cip-nfs`?

Comment (patersonc) [reply scripts/kernel-arch-complete.sh:109]: Yea I thought about that too :P
It's also missing from the cip tree. Will add both in a new commit.

Comment (patersonc) [reply scripts/kernel-arch-complete.sh:109]: There are lots of other omissions, and not just for CIP.
For example there aren't many ltp test plans being reported on. Is this intentional?

Comment (patersonc) [reply scripts/kernel-arch-complete.sh:115]: Mostly intentional. I've just updated the CIP configurations for now.

Comment (gctucker) [reply scripts/kernel-arch-complete.sh:115]: Thanks (as discussed on IRC).

**Review by gctucker — APPROVED** [review]
I'll just wait until the next round of jobs are run on staging to confirm there's no syntax error in the script, then this should be ready to be merged.

**Comment by gctucker** [comment]
OK test reports were sent as expected.

**Comment by gctucker** [comment]
This should get rolled out on Monday and some reports will be sent as soon as a new revision is build and tested on these trees.

**Pull request merged**